Content.Server/MassMedia/Systems/NewsSystem.cs

```diff
@@ -131,6 +134,8 @@ public void OnWriteUiShareMessage(EntityUid uid, NewsWriteComponent component, N
             }
         }
 
+        _audio.PlayPvs(_audio.GetSound(component.ConfirmSound), uid);
```

Comment:
I'm pretty sure you can just use `component.ConfirmSound` without using `GetSound` method here